src/ivory-px4-hw/test-client/calibration/calibrate.py

```diff
@@ -0,0 +1,173 @@
+#! /usr/bin/env python
+
+#  Copyright (C) 2010 Antoine Drouin
+#
+# This file is part of Paparazzi.
+#
+# Paparazzi is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Yeah, particularly if we aren't modifying it at all, we should just link to it. GPL stuff gives Jodee a headache so we should avoid it if at all possible.